internal/cfgplugins/policyforwarding.go

```diff
@@ -572,3 +572,62 @@ func PolicyForwardingGreDecapsulation(t *testing.T, batch *gnmi.SetBatch, dut *o
 		gnmi.BatchReplace(batch, gnmi.OC().NetworkInstance(deviations.DefaultNetworkInstance(dut)).Config(), ni1)
 	}
 }
+
+func MPLSStaticLSPScaleConfig(t *testing.T, dut *ondatra.DUTDevice, ni *oc.NetworkInstance, nexthops []string, labels []int, nexthopsIpv6 []string, labelsforIpv6 []int, ocPFParams OcPolicyForwardingParams) {
+	if deviations.StaticMplsUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			var mplsStaticLspConfig string
+			var mplsStaticLspConfigV6 string
```

Comment:
You can combine like-type var declarations into a single line.

```suggestion
			var mplsStaticLspConfig, mplsStaticLspConfigV6 string
```